src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestPartials.qs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR
+{
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation TestPartials () : Bool
+    {
+        let rotate = Rz(0.25, _);
```

Comment:
It would be nice to add a couple more scenarios here; e.g. for an operation that usually take a nested tuple Op(a, (b, c)) I'd test the cases Op(_, (b, c)), Op(a, (_, c)), Op(a, (_,_)), and Op (a, _).